metricbeat/module/kafka/consumergroup/consumergroup_integration_test.go

```diff
@@ -40,19 +40,19 @@ const (
 )
 
 func TestData(t *testing.T) {
-	t.Skip("Flaky test: https://github.com/elastic/beats/issues/42808")
 	service := compose.EnsureUp(t, "kafka",
 		compose.UpWithTimeout(600*time.Second),
 		compose.UpWithAdvertisedHostEnvFileForPort(9092),
 	)
+	hostForPort := service.HostForPort(9092)
```

Comment:
The variable should probably just be `host`, `HostForPort` means we give a port and get a host